src/_tutorials/language/futures.md

```diff
@@ -11,22 +11,41 @@ nextpage:
 <div class="mini-toc" markdown="1">
   <h4>What's the point?</h4>
 
-  * Dart is single-threaded.
-  * Synchronous code can make your program freeze.
-  * Use `Future` objects (_futures_) to perform asynchronous operations.
-  * Use `await` in an async function to suspend execution until a future completes.
-  * Or use the `then()` method.
-  * Use try-catch expressions in async functions to catch errors.
-  * Or use the `catchError()` method.
-  * You can chain futures to run asynchronous functions in order.
+  * Dart code runs straight through, in a single "thread" of execution.
```

Comment:
I find that description confusing, because it seems to ignore "async" completely.

I think it's mostly the "straight through" phrase which doesn't match the way I think of an event loop.

I would rather say that:

> The Dart *language* is single-threaded. It has no concept of *concurrency*. A Dart program only executes one thing at a time, and code which blocks will block the entire program.

... then move onto isolates, which allow running multiple independent Dart programs concurrently, and async, which allows interleaving code inside a single isolate.

(Still not great, I admit).

Reply:
Thanks for the comments, @lrhn. I'm worried about equating isolates with programs, but I'm working on some wording that might work...